src/main/java/dk/easv/helpers/DocumentHelper.java

```diff
@@ -0,0 +1,22 @@
+package dk.easv.helpers;
+import dk.easv.be.Document;
+import dk.easv.bll.DocumentLogic;
+
+import java.sql.SQLException;
+import java.util.List;
+
+
+public class DocumentHelper {
+    public static StringBuilder convertToString() throws SQLException {
```

Comment:
return string, add the list as an argument use this only for translation to string do not fetch the list from here

Reply:
adding the List as an argument is not working, as the List is returned as NULL somehow and it's giving an error when logging in